apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerDataAdapter.cs

```diff
@@ -155,6 +161,11 @@ IDbCommand IDbDataAdapter.UpdateCommand
         }
 
         /// <summary>
+        /// The set of primary keys defined for <see cref="AutoGeneratedCommandTable"/>.
+        /// </summary>
+        public HashSet<string> AutoGeneratedCommandPrimaryKeys { get; } = new HashSet<string>();
```

Comment:
The user does not need to know it is a `HashSet<string>`. The `ISet<string>` interface would do.